docs/API_reference.md

```diff
@@ -275,6 +276,23 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+---
+
+##### plenty_api_get_property_selections: <a name='get-property-selections'></a>
+
+Return a mapping of Selection values for selction properties.
+
+[*Optional parameter*]:
+
+The **refine** field can be used to reduce the request to specific property IDs.
+example: refine={'propertyID': 123}
+
+[*Output format*]:
+
+There are currently two supported output formats: 'json' and 'dataframe'.  
```

Comment:
You actually only support json, currently.
You could implement dataframe from what you got like this:
dict version:
`{1: {11: {'de': 'value_1', 'en': 'value_2'}}, 2: {12: {'de': 'value_3', 'en': 'value_4'}}}`
dataframe version:
`DataFrame([[1, 11, 'de', 'value_1'], [1, 11, 'en', 'value_2'], [2, 12, 'de', 'value_3'], [2, 12, 'en', 'value_4']], columns=['property_id', 'selection_id', 'lang', 'value'])`